iris-mpc/src/services/processors/batch.rs

```diff
@@ -245,9 +276,6 @@ impl<'a> BatchProcessor<'a> {
                 .collect::<Vec<_>>()
         );
 
-        // Increment batch_id for the next batch
-        self.current_batch_id_atomic.fetch_add(1, Ordering::SeqCst);
```

Comment:
Is this intentional? I haven’t seen what replaces this increment.

Reply:
`self.current_batch_id_atomic` could be removed, and the loop increments `self.batch_sync_shared_state.batch_id += 1` instead, in the block `// Update the shared state with our current state`.

Reply:
It has been changed to be updated after results processing https://github.com/worldcoin/iris-mpc/pull/1572/files#r2276130781